src/main/sensors/battery.c

```diff
@@ -84,9 +83,9 @@ static void updateBatteryVoltage(void)
         vBatFilterIsInitialised = true;
     }
 
-    #ifdef USE_ESC_TELEMETRY
-    if (batteryConfig->batteryMeterType == BATTERY_SENSOR_ESC && isEscTelemetryActive()) {
-        vbatLatest = getEscTelemetryVbat();
+    #ifdef USE_ESC_SENSOR
+    if (feature(FEATURE_ESC_SENSOR) && batteryConfig->batteryMeterType == BATTERY_SENSOR_ESC) {
```

Comment:
I still think this should be selected based on a config parameter, and not based on the feature being enabled, to give users the choice to use ESC feedback current reporting, but use the more accurate FC voltage reporting.